Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -4,7 +4,7 @@
 
 import UIKit
 import Shared
-
+import ComponentLibrary
 protocol SearchEnginePickerDelegate: AnyObject {
```

Comment:
Missing space, will trigget swiftlint otherwise
```suggestion
import ComponentLibrary

protocol SearchEnginePickerDelegate: AnyObject {
```